lib/InitializeJavaAppClass.js

```diff
@@ -215,9 +225,26 @@ module.exports = class {
         }
     }
 
+    __initializeJenkinsFile() {
+        const args = [
+            `${this.cwd}/.pipeline/node_modules/nr-pipeline-ext/lib/templates/pipeline/Jenkinsfile`,
+            `${this.cwd}/`,
+        ]
+        spawnSync('cp', args, { encoding: 'utf-8' })
```

Comment:
don't we need error handling here? like if copy command won't exec correctly causing of permission or etc...